app/views/jobs/show.html.erb

```diff
@@ -16,9 +16,7 @@
 
   <% @job.taggings.each do |tag| %>
     Tag: <%= tag.tag.title  %>
-    Count: <%= tag.tag.tag_count %>
+    Count: <%= tag.tag.tag_count %> (<%= tag.tag.average_salary %>)
```

Comment:
Would have liked to ave seen you used the relationships rather than explicitly using the join table.